### PR TITLE
[explore] Update error messages

### PR DESCRIPTION
**Pull request by chejennifer**
- show topics in the error message (only for cases where the only tile returned is a place overview tile)
- put in the url for the response form
- also fixed css problem with sticky search bar

Screenshot 2023-08-30 at 10 39 50 AM

### CODE REVIEW

File: static/js/apps/explore/success_result.tsx [hunk 1]
```diff
@@ -56,11 +56,28 @@ interface SuccessResultPropType {
...
+      for (const column of block.columns) {
+        for (const tile of column.tiles) {
+          if (tile.type !== "PLACE_OVERVIEW") {
+            return false;
+          }
+        }
+      }
+    }
+  }
+  return true;
```
Comment (pradh) [inline static/js/apps/explore/success_result.tsx:72]: nit: should we do:
if (#categories == 1 && #block-in-category == 1 && #column-in-block == 1 && #tile-in-column == 1 && tile.type == "PLACE_OVERVIEW") return true;

?

**Review by pradh — APPROVED** [review]
Nice!

File: static/js/apps/explore/success_result.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/apps/explore/success_result.tsx]: oh, that's nicer, yes, updated!

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**